content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md

```diff
@@ -1,115 +1,210 @@
+If you are in any doubt about the state of the old DB node, it is recommended to ensure that its VM/container has been deleted.
 
-```bash  
-   cfy_manager dbs remove -a\--address <node's address> -i\--node-id <Node's cloudify node id>
+Next, the DB must be removed from the DB cluster using the following command on any current DB node:
+```bash
+  cfy_manager dbs remove -a <DB address as it appears in the node_ip field of the cfy_manager dbs list>
 ```
 
-* If a database node becomes unhealthy according to the cluster status, it can be re-initialized by running the following command:  
+Then, on every manager the following command must be run:
+```bash
+  cfy_manager dbs remove -a <DB address as it appears in the node_ip field of the cfy_manager dbs list>
+```
 
-```bash  
-   cfy_manager dbs reinit -a\--address <node's address>
+Following this, the manager state should be updated by running the following command once with any CLI connected to the cluster:
+```bash
+  cfy db-nodes update
+```
+
+After removing the DB node, you can verify that the DB cluster settings are correct by checking the DB nodes list on one DB node and all manager nodes with:
+```bash
+  cfy_manager dbs list
+```
+All expected DB nodes should be listed on the DB node and all managers.
+Then, you can confirm the cluster is healthy with:
+```bash
+  cfy cluster status
 ```
+The status should be healthy. Note that cluster status can take up to ~30 seconds to stabilise.
 
-* For switching the current database master node in the database cluster by running the following command:
+### Add
 
-```bash  
-   cfy_manager dbs set-master -a\--address <node's address>
+To add a new DB node, install the node with the same DB cluster settings, including cluster nodes, with the new node in the node list in its config.yaml.
+
+After install is complete, you will need to add the DB to each manager in the cluster. On every manager, run:
+```bash
+  cfy_manager dbs add -a <IP or resolvable DNS name of new broker>
+```
+
+Following this, the manager state should be updated by running the following command once with any CLI connected to the cluster:
+```bash
+  cfy db-nodes update
+```
+
+After adding the DB node, you can verify that the DB cluster settings are correct by checking the DB nodes list on one DB node and all manager nodes with:
+```bash
+  cfy_manager dbs list
+```
+All expected DB nodes should be listed on the DB node and all managers.
+Then, you can confirm the cluster is healthy with:
+```bash
+  cfy cluster status
+```
+The status should be healthy. Note that cluster status can take up to ~30 seconds to stabilise.
+
+### Set master
+
+If you wish to change the current DB master node, e.g. because the current master node is going to be undergoing maintenance operations, run the following command on a DB node:
+```bash
+  cfy_manager dbs set-master -a <intended new master's DB address as it appears in the node_ip field of the cfy_manager dbs list>
```

Comment:
@geokala I think its worth to mention that manager should be in maintenance mode before running this command, because I tried it and I got the following error message
```
2020-12-13 10:21:48,354 - [CONFIG] - INFO - Loading configuration from /etc/cloudify/config.yaml
2020-12-13 10:21:49,610 - [POSTGRESQL_SERVER] - INFO - 172.22.0.11 is async replica. Only sync replicas can become master.
2020-12-13 10:21:49,610 - [POSTGRESQL_SERVER] - INFO - Attempting to promote 172.22.0.11 to sync replica.
2020-12-13 10:22:02,207 - [MAIN] - ERROR - DBManagementError: 172.22.0.11 did not manage to become synchronous replica. Before you retry, please ensure the DB cluster is healthy and the managers are in maintenance mode.
DBManagementError: 172.22.0.11 did not manage to become synchronous replica. Before you retry, please ensure the DB cluster is healthy and the managers are in maintenance mode.
```

Reply:
This happened when trying to make async replica as master node but it failed, so what I did after that is to run change the ip address for sync replica and the leader get changed as expected

Reply:
I thought that should retry- poke me on Monday and I'll check the code.

As for mentioning that the managers should be in maintenance mode, that's in a note at the top of the document.